src/EmailService.Models.Db/DbModelSetting.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;

```

Comment:
maybe rename the file to DbModuleSetting.cs

Reply:
fixed in develop